elements/pfe-collapse/pfe-collapse-panel.ts

```diff
@@ -45,15 +45,19 @@ export class AnimationEndEvent extends ComposedEvent {
 export class PfeCollapsePanel extends LitElement {
   static readonly styles = [style];
 
+  /**
+   * @readonly
+   * Use the toggle method available on pfe-collapse and pfe-collapse-toggle to manually trigger expand.
```

Comment:
Why should this be considered readonly? doesn't setting `el.expanded = true` work?

prior art: 
- https://github.com/material-components/material-web/blob/84857457f0e5daa69744c17675f32ce5403512ec/packages/dialog/mwc-dialog-base.ts#L90
- https://shoelace.style/components/dialog?id=properties
- https://github.com/microsoft/fast/blob/256fe07122c2838e6bf0edd639a539b9ceeaa1bf/packages/web-components/fast-foundation/src/accordion-item/accordion-item.ts#L41-L49

Reply:
I pushed a commit to feat/pfe-tools which moves the sides effects of `toggle()` into `_expandedChanged` and also adds  public `expand()` and `collapse()`. That allows `expanded = true` to operate like it's cousins linked above without breaking stuff.

PTAL

Reply:
Updated to reflect the new expand functionality.